plugin/storage/es/spanstore/reader_test.go

```diff
@@ -307,6 +328,8 @@ func TestSpanReader_GetTrace(t *testing.T) {
 			}, nil)
 
 		trace, err := r.reader.GetTrace(context.Background(), model.NewTraceID(0, 1))
+		require.NotEmpty(t, r.exporter.GetSpans(), "Spans recorded")
```

Comment:
@yurishkuro after workaround, I am still getting Empty spans. Any help?

Reply:
L118:
```
	tracer, exp, closer := tracerProvider()
	logger, logBuffer := testutils.NewLogger()
	r := &spanReaderTest{
		client:    client,
		logger:    logger,
		logBuffer: logBuffer,
		exporter:  exp,
		err:       closer(), // what do you suppose happens here?
```

Reply:
thanks much. that got called and shutdown the tracer.